src/CoffeeMatch.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const CoffeeMatch = () => {
```

Comment:
You'll need a prop here:

```suggestion
const CoffeeMatch = ({matches}) => {
```